documentation/docs/release-notes/3.1.0.md

```diff
@@ -0,0 +1,170 @@
+- troubleshoot issues effectively
+- execute database management tasks seamlessly, whether your databases are on-premises or in the cloud
+
+## Release summary
+
+PMM 3.1.0 enhances MongoDB monitoring with improved query analytics, better visualization, and streamlined troubleshooting. This release also refines role-based access for advisors and alerts, along with various usability improvements and bug fixes.
+
+## Release highlights
+
+### Enhanced MongoDB monitoring
+We've improved MongoDB monitoring with detailed query metrics, better dashboard visualization, and new capabilities to quickly identify performance bottlenecks.
+
+#### Extended metrics and labels for MongoDB Query Analytics (QAN)
+
+The [QAN dashboard](../use/qan/index.md) includes new metrics and filters that highlight inefficient execution paths and offer detailed lock and storage metrics. With the improved filters, you can quickly pinpoint problematic queries based on application, client, user, or plan summary:
+
+##### New metrics
+In the **Query Details** section of the QAN dashboard you'll now find these additional metrics:
+
+=== "Query execution metrics"
+    New metrics for identifying inefficient query patterns:
+
+    - **Docs Examined**: the number of documents scanned. High values may indicate missing or inefficient indexes.
+    - **Keys Examined**: the number of index keys scanned. Compare this with **Docs Examined** to assess index efficiency.
+    - **Docs Returned**: the number of documents returned to the PMM Client. A high **Docs Examined** to **Docs Returned** ratio may indicate inefficiencies that need optimization. 
+
+=== "Response metric"
+    New metric for evaluating query response efficiency:
+
+    **Response Length**: measures the size of the response payload in bytes, helping identify large result sets that may impact performance.
+
+=== "Lock statistics metrics"
+    New metrics for detecting lock contention issues:
+
+    - **Locks Global Acquire Count Read/Write Shared**: tracks how often global locks are used, helping to spot delays caused by multiple processes accessing the same data.
+    - **Locks Database Acquire Count Read Shared**: tracks how often read locks are used in the database.
+    - **Locks Database Acquire Wait Count Read Shared**: counts the times a database read lock request had to wait. High values may signal delays due to contention.
+    - **Locks Database Time Acquiring Micros Read Shared**: captures the time spent waiting for database read locks, critical for performance analysis.
+    - **Locks Collection Acquire Count Read Shared**: tracks how often read locks are used at the collection level.
+    - **Full Scan (COLLSCAN)**: tracks instances where the entire dataset is scanned, meaning every document in the collection is read. This results in high I/O operations, long execution times, and scalability challenges. High  COLLSCAN should be addressed to enhance performance.
```

Comment:
I don't think that it should be under Lock statistics, better to move it to "Query execution metrics"